### PR TITLE
Convert standup plugin to read external config

### PR DESCRIPTION
**Pull request by rattboi**
All configuration was hardcoded for the first version. This
externalizes it to a yaml file that can be set in errbot's config.py,
under STANDUP_CONFIG_PATH, or it defaults to '/etc/standup/config.yaml'.

Also added a requirements.txt that will be auto-installed by errbot
when the plugin is loaded the first time.

Signed-off-by: Bradon Kanyid <bradon@kanyid.org>

### CODE REVIEW

File: err-standup/requirements.txt [hunk 1]
```diff
@@ -0,0 +1,2 @@
+pytz==2016.7
+PyYAML==3.12
```
Comment (omgjlk) [inline err-standup/requirements.txt:2]: Is there a reason to hardcode the versions here?

**Review by omgjlk — APPROVED** [review]
This looks okay, is there a related change that puts down the config file?

File: err-standup/requirements.txt (hunk 1 above)
Comment (rattboi) [reply err-standup/requirements.txt:2]: I just did a 'pip freeze > requirements.txt'. What's a better approach? I'm pretty new at creating full packages in python still.

**Comment by rattboi** [comment]
@j2sol yes, it's in hoist, and was approved yesterday: https://github.com/BonnyCI/hoist/pull/344

**Pull request merged**